speech-to-text/recognize-file.js

```diff
@@ -111,7 +111,15 @@ module.exports = function recognizeFile(options) {
   if (typeof options.file === 'string') {
     fetch(options.file)
       .then(function(response) {
-        return response.readable();
+        // old behavior https://github.com/bergos/nodeify-fetch/blob/v1.0.1/lib/patch-response.js#L23
+        //return response.readable();
+        // new behavior https://github.com/bergos/nodeify-fetch/blob/v2.2.2/lib/patchResponse.js
+        // seems like we just have to check if the body is readable
+        if (response.body.readable) {
+          return response.body
```

Comment:
we should use semicolons here. we'll need to check and make sure we are linting on pre-commit

Reply:
did a `npm run lint` and it didn't find anything. proceeded to add semicolons in the next commit manually.